misc/prtags.txt

```diff
@@ -0,0 +1,29 @@
+cpu             CPU backends
```

Comment:
Like the move to separate this file from the code, in the meantime, do you think it would be more straightforward to make this a JSON file than a plain txt? Thus the pre-processing and loading processes in check_pr_title.py and make_changelog.py could be simplified to just `prtags = json.load(f)`

Reply:
Good idea! Although we only need to store a very basic str to str map.